files/requirements.sh

```diff
@@ -52,7 +52,7 @@ fi
 echo "Using constraints file: ${constraints}"
 
 get_pip_tmp="/tmp/get-pip.py"
-pip_version="19.0.2"
+pip_version="19.2.3"
```

Comment:
What happens if the SDK is installed using pip 9.0.3, which is installed for Python 2.6?

Reply:
The SDK don't support Python 2.6: https://github.com/vmware/vsphere-automation-sdk-python#abstract
It's the same for pyVmomi.

Reply:
Do we know why the newer version of pip is required to install the SDK? Is it working around a bug in the SDK, or is it a legitimate bug in pip that was fixed?

Reply:
It looks like that bug fix was included in pip 19.1.